sdk/storage/storage-file/test/fileclient.spec.ts

```diff
@@ -282,7 +283,7 @@ describe("FileClient", () => {
 
     let eventTriggered = false;
     try {
-      const aborter = Aborter.none;
+      const aborter = new AbortController();
```

Comment:
I know you're not looking at this test specifically, but it doesn't seem to be checking what it says it is. The `abortSignal` is never passed to the `download` method, and if it was cancelled successfully I'd expect an `AbortError` to have been thrown.

Reply:
You're right.
I think the test title is misleading, it only checks for the progress event and doesn't care about aborting.

Reply:
There should some thing missing when moving Aborter parameter to optional options. So here we need to put aborter into fileClient.download options.

Reply:
Thanks @XiaoningLiu, fixed it!